### PR TITLE
MF-93 - Add ingress rules for groups (things and users)

### LINKED ISSUE #93: Add ingress for things groups
There are endpoints in Things and Users service to create groups but currently ingress is only configured for Users groups. 
We can probably use: 
```
/groups/users
/groups/things
```
or
```
/groups-users
/groups-things
```

### PR DESCRIPTION
**Pull request by blokovi**
Signed-off-by: Ivan Milosevic <iva@blokovi.com>

Resolves #93

### CODE REVIEW

File: charts/mainflux/templates/ingress.yaml [hunk 1]
```diff
@@ -71,6 +71,20 @@ spec:
                 name: {{ .Release.Name }}-things
                 port:
                   number: {{ .Values.things.httpPort }}
+          - path: /groups/users.*
```
Comment (drasko) [inline charts/mainflux/templates/ingress.yaml:74]: This is only groups of users? Do we handle groups of things as well?

Comment (blokovi) [reply charts/mainflux/templates/ingress.yaml:74]: @drasko Yes we do, in next rule

**Review by mteodor — APPROVED** [review]

File: charts/mainflux/templates/ingress.yaml (hunk 1 above)
Comment (drasko) [reply charts/mainflux/templates/ingress.yaml:74]: Ah yes, sorry :)

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**